book/website/project-design/sdp.md

```diff
@@ -0,0 +1,20 @@
+(pd-sdp)=
+
+# Sensitive Data Projects
+
+## Prerequisites
+
+| Prerequisite |  Importance  | 
+| ------------ |------------- |
+| {ref}`cl-github-novice` | Helpful |
+
+## Summary
+
```

Comment:
This all looks great! I suggest adding in a sentence defining sensitive data as well, for example something like "Sensitive data must be handled in compliance with General Data Protection Regulation (GDPR), and refers to information regarding: an individual; animals and plants; confidential commercial data; or data that may cause public harm if released."

Reply:
Thank you so much @beckigreen for taking time to review this! 💖 Super suggestion, and I really like your definition. @EstherPlomp has written about Personal data in more detail, so I'm wondering if I should link to this rather than provide a definition. What do you think @EstherPlomp / @KirstieJane?

Reply:
I think I need to read this before providing a decent comment on that! I hope to have some time this week for that, sorry for the slow response!